recipes-extended/xen/files/libxl-iso-hotswap.patch

```diff
@@ -125,9 +124,9 @@ Index: xen-4.9.1/tools/libxl/libxl_disk.c
 +    disk_empty.vdev = libxl__strdup(NOGC, olddisk->vdev);
 +    disk_empty.pdev_path = libxl__strdup(NOGC, "");
 +    disk_empty.is_cdrom = 1;
-+    libxl__device_disk_setdefault(gc, &disk_empty, domid);
++    libxl__device_disk_setdefault(gc, &disk_empty, stubdomid);
```

Comment:
Are these stubdomid changes correct?  Is this to force the CDROM VBDs to the stubdom?  The entire commit messages is replicated in the libxl-remove-useless-nodes.patch description, so it's not immediately clear why these are needed.

Will this break the non-stubdom case?